pkg/json.go

```diff
@@ -82,6 +83,18 @@ type JSONScorecardResultV2 struct {
 	Metadata       []string            `json:"metadata"`
 }
 
+// JSONDependencydiffResult exports dependency-diff results as JSON for new detail format.
+type JSONDependencydiffResult struct {
+	ChangeType           *ChangeType           `json:"changeType"`
```

Comment:
Let's add a comment + issue to update these fields to be consistent with OSV naming convention.